### DISCUSSION TITLE
What models are compatible with fooocus? is there a list? Or how can I determine if the model or lora will work?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#3611] [General]
What models are compatible with fooocus? is there a list? Or how can I determine if the model or lora will work?

**Comment by Rogala** [comment]
[~99% Checkpoint, ~95% LoRA, 50/50 Refiner SD1.5](https://civitai.com/models)

![image](https://github.com/user-attachments/assets/cf35ca7d-a551-4b98-aa60-8b971c7d6ec7)

**Comment by Yogesh-DevHub** [comment]
You can get the models from [civitai](https://civitai.com/models). Apply a filter, SDXL to see the compatible models.

Here is a short video explaining models and lora in fooocus UI: https://youtu.be/baU1t95lDNg?si=xV9GZ6IK-_KOGOts
